### ISSUE TITLE
Explore install script for datakit

### ISSUE THREAD

**Issue by tthibo** [associatedpress/datakit-core#12]
In addition to improving our documentation and creating some intro tutorials, we discussed the possibility of creating an install script (a la https://get.rvm.i).

**Comment by zstumgoren** [comment]
Datakit is designed to use standard Python package installation techniques (i.e. `pip install` or `pipenv install`). So basic installation of the core library and plugins should be straight-forward.

**However**, the level of complexity associated with *configuring* an ecosystem of datakit plugins for a particular individual or team can vary depending on the plugins used. @tthibo - Is the goal here to create a higher-level script that can install and help configure a common set of plugins? Or perhaps multiple scripts designed for different user types (e.g. Ruby full stack web developer vs. R data analyst vs. Python data analyst)?

**Comment by tthibo** [comment]
Yep, I was thinking more along the lines of a basic setup for a reasonable running rig. Looking for a way to give folks a quickstart that they can later configure. Perhaps a "common set of plugins" is the best way to start, and then we could always build out different setup scripts for different use cases.

**Comment by zstumgoren** [comment]
@tthibo Yep. that makes total sense. That was actually the reason behind the datakit-ap repo. Basically a way to minimize some of the pain of installing the ecosystem and configuring it out of the gates. One starting point might be to simply clean up, generalize and open source datakit-ap under a new name. But open to other ideas as well. That plus some related documentation on how to use and customize.
